packages/sdks-tests/src/snippet-tests/integrating-sections.spec.ts

```diff
@@ -12,7 +12,7 @@ test.describe('Integrating Sections', () => {
     await page.goto('/announcements/foo');
 
     await expect(page.locator('body')).not.toHaveText('This is an announcement banner');
-    await findTextInPage({ page, text: 'Announcement Bar not Found' });
```

Comment:
i don't this test is meant to check for `rest of the content`. If you see we already test it inside a test below